engines/mus.cpp

```diff
@@ -157,7 +157,12 @@ namespace pono {
           id = lhs;
         }
       }
-      transIdToConjunct.insert({id, unrollUntilBound(tc, k)});
+      Term t = unrollUntilBound(tc, k);
+      if (options_.mus_ignore_yosys_internal_netnames_ && id->to_string().rfind('$', 0) == 0) {
```

Comment:
This looks great. A small change you might consider is to move the `id->to_string().rfind('$', 0) == 0` into its own function. Two reasons:

1. The function name will help document what the code is doing. This can be inferred from context here, but is even clearer when it is moved into a function I think.
2. If (god forbid) the way Yosys represents internal identifiers changes, or we find later that other patterns are also used to describe internal names, we can modify the function and leave everything else the same.

Reply:
Tweaked in [31c9235](https://github.com/GaloisInc/pono/pull/1/commits/31c9235f48ac1f4c7fa896ab24137c0352e300cc)